charts/redis-ha/values.yaml

```diff
@@ -233,6 +233,10 @@ redis:
   ## Ref: https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/
   lifecycle: {}
 
+  ## annotations for the redis statefulset itself (not the individual redis, sentinel, metrics pods)
+  statefulset:
```

Comment:
Change this to just `annotations` under `redis.annotations` rather than `redis.statefulset.annotations`. `podAnnotations` is normally used for pod level annotations, `annotations` for top level.